docker/entrypoint.sh

```diff
@@ -5,6 +5,9 @@
 # Make non-zero exit codes & other errors fatal.
 set -euo pipefail
 
+mkdir -p /root/.config/adr
+cp docker/config.toml /root/.config/adr/config.toml
```

Comment:
You can do these directly in the `Dockerfile`.
You can instead set an `ADR_CONFIG_PATH` env variable to point to `/app/docker/config.toml`.
I find it more clear than using the default path that adr chooses.

Reply:
Yeah, that's better.  I did that instead.  Thanks!